modules/gltf/gltf_document.cpp

```diff
@@ -6565,11 +6746,20 @@ void GLTFDocument::_convert_animation(Ref<GLTFState> p_state, AnimationPlayer *p
 		if (!animation->track_is_enabled(track_i)) {
 			continue;
 		}
-		String orig_track_path = animation->track_get_path(track_i);
-		if (String(orig_track_path).contains(":position")) {
-			const Vector<String> node_suffix = String(orig_track_path).split(":position");
+		String final_track_path = animation->track_get_path(track_i);
+		Node *animation_base_node = p_animation_player->get_parent();
+
+		if (final_track_path.begins_with("%")) {
```

Comment:
Made this comment in discord too, but this special case should not be necessary. `get_node_or_null` should handle this already.